components/utils/lazysections.tsx

```diff
@@ -21,16 +21,22 @@ function useOnScreen(ref, rootMargin) {
       observer.observe(currentRef)
     }
     return () => {
-      observer.unobserve(currentRef)
+      if (currentRef) {
+        observer.unobserve(currentRef)
+      }
```

Comment:
Working around possible `null`, even though it won't _actually_ be null at this point.  I'm sure there is a better way to lay this effect out so this isn't required.